### ISSUE TITLE
Connected-react-router v6 breaking this app

### ISSUE THREAD

**Issue by deksden** [notrab/create-react-app-redux#75]
Migrating from v4 to v5 (v6) is breaking app. 

Here is FAQ with instruction how to migrate:
https://github.com/supasate/connected-react-router/blob/master/FAQ.md#how-to-migrate-from-v4-to-v5

**Comment by vchouhan** [comment]
"connected-react-router": "^6.2.1",
    "react-redux": "^6.0.0",

Tried running with the above versions of connected-react-router and react-redux and the app keeps throwing the following error.
`Could not find router reducer in state tree, it must be mounted under "router"`

**Comment by dcsan** [comment]
yeah I hit this exact problem. the joys of the react ecosystem thrash, everything breaks 15 minutes later.

**Comment by notrab** [comment]
> yeah I hit this exact problem. the joys of the react ecosystem thrash, everything breaks 15 minutes later.

Not really.. The release is tagged a breaking change. Don't upgrade and you won't have this issue 😛 

The redux router library was always tagged as alpha anyways afaik.

**Comment by notrab** [comment]
@vchouhan have you seen https://github.com/notrab/create-react-app-redux/pull/71 ?

It might be worth checking out that branch, confirming it works and then we can push it into master.
